README.md

```diff
@@ -1 +1,123 @@
-# Leaderboard-App
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Mohammad Yaser Safi <a name="about-project"></a>
+<br>
+<br>
+The Leaderboard Project, created solely by me, showcases my skills in JavaScript ES6, HTML, CSS, and my ability to use Webpack for an efficient development workflow. This web-page demonstrates a simple, user-friendly Leaderboard management system, allowing users to add, complete, and remove tasks dynamically. The project highlights my commitment to continuous learning and independent web development.
+<br>
+<br>
+
+## 🛠 Built With <a name="built-with"></a>
+<br>
+
+
+  <summary>Page was built using HTML, CSS, JavaScript and used Webpack to bundle javascript files<br>
+  </summary>
+
+<br>
+
+### Key Features <a name="key-features"></a>
+With this To Do List project we can create update and delete one or more tasks
+
+<br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<br>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+<br>
+
+A great project which can be used for almost anything, as it is made dynamically, so information can be updated or added through Javascript.
+
+To get a local copy up and running, follow these steps:
+<br>
+- Choose a directory
+- Open a Terminal
+- write git clone https://github.com/MohammadYaser/Leaderboard-App.git
+- get into to directory "Capstone Project"
```

Comment:
- [ ] [OPTIONAL] Kindly edit the readme file to suit the current project. Here the key features should be API calls (post/get requests), refresh button, etc.


![learderboard](https://github.com/MohammadYaser/Leaderboard-App/assets/53124548/f41939ae-7427-4d26-b5aa-490148f1514e)